### DISCUSSION TITLE
CPCM-X and hess?

### DISCUSSION THREAD

**Discussion by metma99** [grimme-lab/xtb#1038] [Q&A] [answered]
Hello:
I am aware that cpcm-x is bleeding edge. I tried to run a frequency calculation and cpcm-x with xtb-6.7.0  but the solvent part of the input was completely ignored by the calculation. However, the hess function is turned on if I use alpb.
Are you planning to implement this feature?
Thank you very much for your time,
Markus

**Comment by TyBalduf** [comment, marked as answer]
CPCM-X actually has neither 1st nor 2nd derivatives implemented yet. In principle, numerical derivatives could be added (even the gas phase Hessian is computed numerically), but this would likely be pretty inefficient prior to analytic 1st derivatives being implemented.
